.gitignore

```diff
@@ -9,3 +9,5 @@ npm-debug.log
 yarn-error.log
 .DS_Store
 nohup.out
+yarn.lock
```

Comment:
Should `yarn.lock` be gitignored?

Reply:
I honestly don't care either way. I think we have (need to have) everything the library requires in the package.json for a dev to get spun up and start contributing.

Reply:
I've read this some time ago: https://github.com/yarnpkg/yarn/issues/838#issuecomment-253362537